include/DREAM/Equations/Fluid/LCFSLossRateTerm.hpp

```diff
@@ -0,0 +1,51 @@
+#ifndef _DREAM_EQUATION_FLUID_LCFS_LOSS_RATE_TERM_HPP
+#define _DREAM_EQUATION_FLUID_LCFS_LOSS_RATE_TERM_HPP
+
+#include "DREAM/Equations/RunawayFluid.hpp"
+#include "DREAM/Equations/RunawaySourceTerm.hpp"
+#include "FVM/Equation/EquationTerm.hpp"
+#include "FVM/Grid/Grid.hpp"
+#include "FVM/Equation/DiagonalComplexTerm.hpp"
+
+namespace DREAM {
+    class LCFSLossRateTerm : public RunawaySourceTerm, public FVM::DiagonalComplexTerm {
+    private:
+    
+        FVM::UnknownQuantityHandler *unknowns;
+        real_t scaleFactor;
+        real_t* t_loss;
+        const len_t userGivenPsiEdge_t0;
+        real_t psi_edge_t0;
+        const len_t id_psi;
+        real_t* GammaLoss = nullptr;
+        len_t ir_LCFS;
```

Comment:
len_t is unsigned, so if you want to set this variable to a negative value to indicate that there are no closed flux surfaces, you should change the type of this variable to int_t